subsys/bluetooth/mesh/Kconfig

```diff
@@ -296,6 +296,14 @@ config BT_MESH_ADV_STACK_SIZE
 	  NOTE: This is an advanced setting and should not be changed unless
 	  absolutely necessary
 
+config BT_MESH_ADV_PRIO
+	int "Mesh advertiser thread priority"
+	depends on BT_MESH_ADV_LEGACY
+	default 7
+	help
+	  NOTE: This is an advanced setting and should not be changed unless
+	  absolutely necessary
```

Comment:
Some more descriptive text here would probably help users determine whether it's necessary to change this

Reply:
In fact, I don’t know why the priority is set to 7. 😂

Reply:
Heh, that's fair. The help text usually contains a description of what this setting does though, which is all I'm requesting